app/jobs/disability_claim/document_upload.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 class DisabilityClaim::DocumentUpload < ActiveJob::Base
-  # TODO (AJM): Pass in user once it can be serialized using GlobalID
-  def perform(filename, claim_id, tracked_item_id)
-    client = EVSS::DocumentsService.new(User.sample_claimant)
+  def perform(filename, user_attrs, claim_id, tracked_item_id)
+    user = User.new(user_attrs)
+    client = EVSS::DocumentsService.new(user)
```

Comment:
Maybe it could look like 

```ruby
user = Session.whatever
headers = EVSS::VaafiHeader.new(user)
client = EVSS::ClaimService.new(headers)
```

And for the active job, the headers class can know how to (de)serialize itself.